addons/base_setup/models/res_config_settings.py

```diff
@@ -40,6 +46,7 @@ class ResConfigSettings(models.TransientModel):
             implied_group='base.group_multi_currency',
             help="Allows to work in a multi currency environment")
     external_report_layout_id = fields.Many2one(related="company_id.external_report_layout_id", readonly=False)
+    external_report_layout = fields.Selection(selection=EXTERNAL_LAYOUTS, string="External Layout", compute='_compute_external_report_layout', inverse='_inverse_external_report_layout')
```

Comment:
seems overkill.  :suspect: 

Did you try with a domain with [('id', 'in', [ref(), ref(), ref(), ref()])] ? :thinking: